ChangeLog.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## Changes between 0.2.0 and 0.3.0
+
+* Added a new `bifrost.core/handler` function that works like
+  `bifrost.core/interceptor`, but assocs an error response if the channel is
+  already closed.
+* Removed the lower-level `bifrost.core/async-interceptor` function.
```

Comment:
Really appreciate the changelog entry, would you be opposed to moving this item to the top of the list prefixed with a **[BREAKING]**?

Reply:
Yep, good plan